pandas/core/arrays/categorical.py

```diff
@@ -1531,7 +1531,7 @@ def check_for_ordered(self, op):
     def _values_for_argsort(self):
         return self._codes.copy()
 
-    def argsort(self, *args, **kwargs):
+    def argsort(self, ascending=True, kind='quicksort', *args, **kwargs):
         # TODO(PY2): use correct signature
```

Comment:
can remove this comment? is the doc-string updated?

Reply:
Yeah. I was going to do Categorical as a followup, but it seems tiny so I'll do it here.